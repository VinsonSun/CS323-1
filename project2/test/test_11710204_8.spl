struct Apple{
    int price;
};
int test_11710204_8()
{
  int a = 2;
  struct Apple apple;
  apple.price = a;
  return apple;
}
