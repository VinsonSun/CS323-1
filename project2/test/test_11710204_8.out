Error type 8 at Line 9: incompatiable return type
